Orange/widgets/classify/tests/test_owsvmclassification.py

```diff
@@ -0,0 +1,110 @@
+                                  (1, [True, True, True]),
+                                  (2, [True, False, False]),
+                                  (3, [True, True, False])):
+            self.widget.kernel_box.buttons[button_pos].click()
+            self.widget.kernel_box.buttons[button_pos].isChecked()
+            self.assertEqual([not self.widget.spin_boxes[i].box.isHidden() for i in range(2, 5)], value)
+
+    def test_optimization_box_visible(self):
+        """ Check if both spinner box is visible after starting widget """
+        self.assertEqual(self.widget.spin_boxes[5].box.isHidden(), False)
+        self.assertEqual(self.widget.spin_boxes[6].box.isHidden(), False)
+
+    def test_optimization_box_checked(self):
+        """ Check if spinner box for iteration limit is enabled or disabled """
+        for value in (True, False):
+            self.widget.max_iter_check_box.setChecked(value)
+            self.assertEqual(self.widget.max_iter_check_box.isChecked(), value)
+            self.assertEqual(self.widget.spin_boxes[6].isEnabled(), value)
+
+    def test_type_button_checked(self):
+        """ Check if SVM type is selected after click """
+        self.widget.type_box.buttons[0].click()
+        self.assertEqual(self.widget.type_box.buttons[0].isChecked(), True)
+        self.widget.type_box.buttons[1].click()
+        self.assertEqual(self.widget.type_box.buttons[1].isChecked(), True)
+
+    def test_type_button_properties_visible(self):
+        """ Check if spinner box in SVM type are visible """
+        self.assertEqual(not self.widget.spin_boxes[0].isHidden(), True)
+        self.assertEqual(not self.widget.spin_boxes[1].isHidden(), True)
+
+    def test_data_before_apply(self):
+        """ Check if data are set """
+        self.widget.set_data(Table("iris")[:100])
+        self.widget.apply()
+        self.assertEqual(len(self.widget.data), 100)
+
+    def test_output_signal_classifier(self):
+        """ Check if we have classifier on output """
+        output_classifier = False
+        for signal in self.widget.outputs:
+            if signal.name == "Classifier":
+                output_classifier = True
+                break
+        self.assertEqual(output_classifier, True)
+        self.assertEqual(self.widget.learner, None)
+
+    def test_output_signal_learner(self):
+        """ Check if we have on output learner """
+        self.widget.kernel_box.buttons[0].click()
+        self.widget.set_data(Table("iris")[:100])
+        self.widget.apply()
+        self.assertNotEqual(self.widget.learner, None)
+
+    def test_output_params(self):
+        """ Check ouput params """
+        self.widget.kernel_box.buttons[0].click()
+        self.widget.set_data(Table("iris")[:100])
+        self.widget.max_iter_check_box.setChecked(True)
+        self.widget.apply()
+        self.widget.type_box.buttons[0].click()
+        self.assertEqual(self.widget.learner.params.get('C'), self.widget.spin_boxes[0].value())
```

Comment:
Corrected. I add variable spin_boxes into set_up function, because I use it at other tests too.